src/synthesizer/components/stellar.py

```diff
@@ -985,128 +985,124 @@ def check_young_old_units(self, young, old):
         return young, old
 
     def plot_spectra(
-            self,
-            show=False,
-            spectra_to_plot=None,
-            ylimits=("peak", 5),
-            figsize=(3.5, 5),
+        self,
+        spectra_to_plot=None,
+        show=False,
+        ylimits=(),
+        xlimits=(),
+        figsize=(3.5, 5),
     ):
         """
-        plots all spectra associated with a galaxy object
+        Plots either a specific spectra or all spectra provided in a dictionary.
+
+        This is a generic plotting function to be used either directly or to be
```

Comment:
This and other docstrings need updating, this is _not_ the generic plotting function

Reply:
🤦 indeed! Now done!